spec/models/service_template_provision_task_spec.rb

```diff
@@ -3,16 +3,16 @@
+      @request   = FactoryGirl.create(:service_template_provision_request, :description => 'Service Request', :userid => @admin.userid)
+      @task_0    = FactoryGirl.create(:service_template_provision_task,    :description => 'Task 0 (Top)', :userid => @admin.userid, :status => "Ok", :state => "pending",  :miq_request_id => @request.id, :request_type => "clone_to_service")
+      @task_1    = FactoryGirl.create(:service_template_provision_task,    :description => 'Task 1', :userid => @admin.userid, :status => "Ok", :state => "pending",  :miq_request_id => @request.id, :request_type => "clone_to_service", :options => {:service_resource_id => FactoryGirl.create(:service_resource, :provision_index => 7, :scaling_min => 1, :scaling_max => 1, :resource_type => 'ServiceTemplate').id})
+      @task_1_1  = FactoryGirl.create(:service_template_provision_task,    :description => 'Task 1 - 1', :userid => @admin.userid, :status => "Ok", :state => "pending",  :miq_request_id => @request.id, :request_type => "clone_to_service", :options => {:service_resource_id => FactoryGirl.create(:service_resource, :provision_index => 1, :scaling_min => 1, :scaling_max => 3, :resource_type => 'ServiceTemplate').id})
+      @task_1_2  = FactoryGirl.create(:service_template_provision_task,    :description => 'Task 1 - 2', :userid => @admin.userid, :status => "Ok", :state => "pending",  :miq_request_id => @request.id, :request_type => "clone_to_service", :options => {:service_resource_id => FactoryGirl.create(:service_resource, :provision_index => 5, :scaling_min => 1, :scaling_max => 1, :resource_type => 'ServiceTemplate').id})
+      @task_2    = FactoryGirl.create(:service_template_provision_task,    :description => 'Task 2', :userid => @admin.userid, :status => "Ok", :state => "pending",  :miq_request_id => @request.id, :request_type => "clone_to_service", :options => {:service_resource_id => FactoryGirl.create(:service_resource, :provision_index => 9, :scaling_min => 1, :scaling_max => 1, :resource_type => 'ServiceTemplate').id})
+      @task_2_1  = FactoryGirl.create(:service_template_provision_task,    :description => 'Task 2 - 1', :userid => @admin.userid, :status => "Ok", :state => "pending",  :miq_request_id => @request.id, :request_type => "clone_to_service", :options => {:service_resource_id => FactoryGirl.create(:service_resource, :provision_index => 2, :scaling_min => 1, :scaling_max => 1, :resource_type => 'ServiceTemplate').id})
+      @task_3    = FactoryGirl.create(:service_template_provision_task,    :description => 'Task 3', :userid => @admin.userid, :status => "Ok", :state => "finished", :miq_request_id => @request.id, :request_type => "clone_to_service", :options => {:service_resource_id => FactoryGirl.create(:service_resource, :provision_index => 3, :scaling_min => 1, :scaling_max => 5, :resource_type => 'ServiceTemplate').id})
```

Comment:
Would be nice to restructure these lines if you are modifying this file.  Some are over 400 chars long and it would sure help readability.